meetings/conferences/serializers.py

```diff
@@ -1,22 +1,8 @@
-from rest_framework import serializers as ser
-from django_countries.fields import CountryField
+from rest_framework_json_api import serializers
 
 from conferences.models import Conference
 
-class ConferenceSerializer(ser.ModelSerializer):
-    country = CountryField() #get country_dict working later
```

Comment:
With `rest_framework_json_api`, its `daserize` in `settings.py` and Ember expecting to auto convert dashes to camelCase then we don't have to specify each and every field because it does it automatically, we only need to specify ones that don't conform to the system or need a bit more input in order to make a sensible field.